l1-contracts/contracts/zksync/interfaces/IExecutor.sol

```diff
@@ -155,18 +155,29 @@ interface IExecutor is IBase {
     function revertBatches(uint256 _newLastBatch) external;
 
     /// @notice Event emitted when a batch is committed
+    /// @param batchNumber Number of the batch committed
+    /// @param batchHash Hash of the L2 batch
+    /// @param commitment Verified input for the zkSync circuit
     /// @dev It has the name "BlockCommit" and not "BatchCommit" due to backward compatibility considerations
     event BlockCommit(uint256 indexed batchNumber, bytes32 indexed batchHash, bytes32 indexed commitment);
 
     /// @notice Event emitted when batches are verified
+    /// @param previousLastVerifiedBatch Batch number of the previous last verified batch
+    /// @param currentLastVerifiedBatch Batch number of the current last verified batch
     /// @dev It has the name "BlocksVerification" and not "BatchesVerification" due to backward compatibility considerations
     event BlocksVerification(uint256 indexed previousLastVerifiedBatch, uint256 indexed currentLastVerifiedBatch);
 
     /// @notice Event emitted when a batch is executed
+    /// @param batchNumber Number of the batch committed
+    /// @param batchHash Hash of the L2 batch
+    /// @param commitment Verified input for the zkSync circuit
     /// @dev It has the name "BlockExecution" and not "BatchExecution" due to backward compatibility considerations
     event BlockExecution(uint256 indexed batchNumber, bytes32 indexed batchHash, bytes32 indexed commitment);
 
     /// @notice Event emitted when batches are reverted
+    /// @param totalBatchesCommitted Total number of committed batches
```

Comment:
better to mention that this is the total number of committed batches after the revert is complete